frontend/src/FirstRun/AuthenticationRequiredModalContentConnector.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
```

Comment:
I'd recommend doing it in another PR  to avoid adding complexity to this one

This PR is mostly made to have a modern 1:1 adaptation of a few components

Reply:
I've moved the behavior of the connector into the modal itself